app/assets/javascripts/application.js

```diff
@@ -33,7 +33,33 @@ $(function() {
 });
 
 function executeAttachinary() {
+  var js_template = [
+    "<ul class='attachinary_images_list'>",
```

Comment:
@daniloisr don't we need a `<%= raw %>` block or something so ERB won't compile it?